spec/feature_steps/zone_steps.rb

```diff
@@ -0,0 +1,3 @@
+def create_zones
```

Comment:
should this be called `create_zones` or `create_zone`?

Reply:
👍 I think maybe `create_default_zone` as that's all that's currently being done / needed...